dumper/app/src/main/resources/oracle-stats/native/table-types-dtl.sql

```diff
@@ -0,0 +1,89 @@
+-- Copyright 2022-2024 Google LLC
+-- Copyright 2013-2021 CompilerWorks
+--
+-- Licensed under the Apache License, Version 2.0 (the "License");
+-- you may not use this file except in compliance with the License.
+-- You may obtain a copy of the License at
+--
+--     http://www.apache.org/licenses/LICENSE-2.0
+--
+-- Unless required by applicable law or agreed to in writing, software
+-- distributed under the License is distributed on an "AS IS" BASIS,
+-- WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+-- See the License for the specific language governing permissions and
+-- limitations under the License.
+SELECT
+  D.con_id "ConId",
+  D.owner "Owner",
+  D.table_name "TableName",
+  D.partitioned "Partitioned",
+  D.iot_type "IotType",
+  D.nested "Nested",
+  D.temporary "Temporary",
+  D.secondary "Secondary",
+  D.cluster_name "ClusterName",
+  D.source "Source",
+  E.partitioning_type "PartitioningType",
+  E.subpartitioning_type "SubpartitioningType",
+  E.partition_count "PartitionCount",
+  F.count "SubpartitionCount"
+FROM (
+  SELECT
+    A.con_id,
+    A.owner,
+    A.table_name,
+    A.partitioned,
+    A.iot_type,
+    A.nested,
+    A.temporary,
+    A.secondary,
+    A.cluster_name,
+    'TAB' source
+  FROM cdb_tables A
+  WHERE A.owner <> 'SYSTEM' AND A.owner NOT LIKE '%SYS'
```

Comment:
I think we should be consistent in all the scripts and decide to exclude SYSTEM objects or not. This filtering is included in this PR but it was not in the previous one that added db_objects and used_space_details.

I'd suggest to simplify the dumper code and remove the filters. We can implement it in the assessment metadata postprocessor.

Reply:
Done, the filters reduce the output size by about 1/3, so it's not a very big difference.

Filtering is a responsibility of Assessment, Dumper can do it where it's not practical to skip the filter, e.g. the results would be incorrect or larger by an order of magnitude.